pyteal/stack_frame.py

```diff
@@ -133,20 +133,16 @@ def _is_pyteal(self) -> bool:
     def _frame_info_is_pyteal(cls, f: FrameInfo) -> bool:
         return bool(cls._internal_paths_re.search(f.filename))
 
-    def _is_pyteal_import(self) -> bool:
-        return self._frame_info_is_pyteal_import(self.frame_info)
-
     @classmethod
     def _frame_info_is_pyteal_import(cls, f: FrameInfo) -> bool:
         cc = f.code_context
         if not cc:
             return False
 
-        code = "".join(cc).split()
+        code = re.split(r"\s|\.", " ".join(cc))
```

Comment:
this splits up an array of strings by spaces and periods

Reply:
will add a comment

Reply:
f94fa7a8f5ee7f5356eafb9f462c192c3a605605